src/System.Collections.Immutable/tests/ImmutableArrayTest.cs

```diff
@@ -292,22 +292,22 @@ public void CovarianceImplicit()
         }
 
         [Fact]
-        public void CreateByCovariantStaticCast()
+        public void CastUpReference()
         {
             ImmutableArray<string> derivedImmutable = ImmutableArray.Create("a", "b", "c");
-            ImmutableArray<object> baseImmutable = ImmutableArray.Create<object, string>(derivedImmutable);
+            ImmutableArray<object> baseImmutable = ImmutableArray<object>.CastUp(derivedImmutable);
```

Comment:
Do the tests have internals access?  If so, you might want to add checks to verify that the underlying array has reference equality.  If not, ignore this.